snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedComponent.java

```diff
@@ -65,6 +66,8 @@ public String getModuleId() {
 	 * @beta - this method is subject to changes or even removal in future releases.  
 	 * @return - the icon ID associated with this component
 	 */
+	@Override
+	@JsonIgnore
```

Comment:
This annotation was removed in a recent commit. I'll remove it after merging this PR to 6.x.